pkg/controller/import-controller.go

```diff
@@ -643,7 +648,7 @@ func createImporterPod(log logr.Logger, client client.Client, image, verbose, pu
 }
 
 // makeImporterPodSpec creates and return the importer pod spec based on the passed-in endpoint, secret and pvc.
-func makeImporterPodSpec(namespace, image, verbose, pullPolicy string, podEnvVar *importPodEnvVar, pvc *corev1.PersistentVolumeClaim, scratchPvcName *string, podResourceRequirements *corev1.ResourceRequirements) *corev1.Pod {
+func makeImporterPodSpec(namespace, image, verbose, pullPolicy string, podEnvVar *importPodEnvVar, pvc *corev1.PersistentVolumeClaim, scratchPvcName *string, podResourceRequirements *corev1.ResourceRequirements, workloadNodePlacement cdiv1.NodePlacement) *corev1.Pod {
```

Comment:
Any particular reason you are dereferencing the workloadNodePlacement above? Instead of passing a pointer as a parameter you are now passing the entire struct. I think you can have workloadNodePlacement *cdiv1.NodePlacement without any code changes in the actual function.